docs/users_guide.md

```diff
@@ -1,128 +1,128 @@
+  - `gcc`
+  - one of the following: dnf download plugin(`dnf-plugins-core`), `yumdownloader` or `rpm-devel`
 
-  If `rpm-devel` has been installed, it is used. If not, the download utiliy is used to download and use the build required RPM packages in installing process.
+  If `rpm-devel` has been installed, then it is used. If not, the download utility is used to download and use the required RPM packages for the build in install process.
 
-  To install dnf download plugin on DNF environment such as Fedora.
-
-  ```
+  To install dnf download plugin in a DNF environment such as Fedora:
+  ``` ShellSession
   # dnf install 'dnf-command(download)'
   ```
 
-  To install `yumdownloader` on Yum environment such as CentOS.
-
-  ```
+  To install `yumdownloader` in a Yum environment such as CentOS:
+  ``` ShellSession
   # yum install /usr/bin/yumdownloader
   ```
 
-  See also installed packages in [Dockerfiles for testing](../ci/).
+  See also the installed packages in [Dockerfiles for testing](../ci/).
 
 
-- Q3. Does this installer install the Python binding on system Python (`/usr/bin/python*`)?
-- A3. No. The installer skips installing it on system Python.
-  It is recommended that you would install it manually from the RPM package(`python{,2,3}-rpm`).
+- Q3. Does this installer install the Python bindings for the system Python (`/usr/bin/python*`)?
+- A3. No. The installer skips installing it for the system Python.
+  It is recommended that you would install it the python bindings using the the RPM package(`python{,2,3}-rpm`) instead for a system wide installation.
 
 
 - Q4. How is `rpm-py-installer` version mapped to `python[23]-rpm` version?
-- A4. The version mapping's default behavior is to use same version with the system RPM's version on local. See [#94](https://github.com/junaruga/rpm-py-installer/issues/94) for detail.
+- A4. The version mapping's default behavior is to use same version as the system RPM's version. See [#94](https://github.com/junaruga/rpm-py-installer/issues/94) for details.
 
-- Q5. Is it possible to install the Python binding of specific version?
-- A5. Yes. Possible. But it may be failed to install. Set version number seeing [RPM release page](https://github.com/rpm-software-management/rpm/releases).
+- Q5. Is it possible to install the Python bindings of a specific version?
+- A5. Yes, it is possible, but it may fail to install. Set the version number (see the [RPM release page](https://github.com/rpm-software-management/rpm/releases) for valid choices) using the environment variable `RPM_PY_VERSION`.
 
-  ```
+  ``` ShellSession
   $ RPM_PY_VERSION=4.13.0 python -c "$(curl -fsSL https://raw.githubusercontent.com/junaruga/rpm-py-installer/master/install.py)"
   ```
+
 - Q6. How to update my Python package's RPM spec file `foo.spec`?
-- A6. You can add below command in your spec file `foo.spec`.
+- A6. You can add the following command to your spec file `foo.spec`:
 
-  ```
+  ``` bash
```

Comment:
What is the difference between `ShellSession` and `bash` for multi lines code format?
I am not familiar with that. Is there the manual document?

Reply:
`bash` is intended for actual bash scripts (= non-interactive use) and `ShellSession` for interactive sessions (at least that is my understanding). The closest that I've found to a documentation is this list of syntax highlighting values: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Reply:
Sure, alright. Thanks for the explanation!